src/test/java/com/houarizegai/calculator/ui/CalculatorUI_CalculatorUI_772bd0bc5d_Test.java

```diff
@@ -41,9 +41,10 @@ void setUp() {
     void testWindowInitialization() {
         window = calculatorUI.window;
         assertNotNull(window, "Window is not initialized");
-        assertEquals(APPLICATION_TITLE, window.getTitle(), "Window title is incorrect");
-        assertEquals(WINDOW_WIDTH, window.getWidth(), "Window width is incorrect");
-        assertEquals(WINDOW_HEIGHT, window.getHeight(), "Window height is incorrect");
+        // assertEquals(APPLICATION_TITLE, window.getTitle(), "Window title is incorrect");
+        // assertEquals(WINDOW_WIDTH, window.getWidth(), "Window width is incorrect");
+        // assertEquals(WINDOW_HEIGHT, window.getHeight(), "Window height is incorrect");
+        //WINDOW_HEIGHT, WINDOW_WIDTH,APPLICATION_TITLE are private fields
```

Comment:
WINDOW_HEIGHT, WINDOW_WIDTH,APPLICATION_TITLE are private fields, should not be there in the test files generated

Reply:
prompt has private fields in it -
 fields:{
            name:WINDOW_HEIGHT,
            type:int,
            accessModifier:private static final,
            value:600
          },fields:{
            name:BUTTON_WIDTH,
            type:int,
            accessModifier:private static final,
            value:80
          },fields:{
            name:BUTTON_HEIGHT,
            type:int,
            accessModifier:private static final,
            value:70
          }